docs: re-add `message validation guide` and change code snippet

animeshk923 opened the pull request:
**Description**

- Re-added the `message validation guide` in the next-major-spec branch.
- Changed the code snippet to match v3.

**Related issue(s)**
Fixes #1865

animeshk923 committed:
re-add `message validation guide`

changes:

- Re-added the `message validation guide` in
the next-major-spec branch.
- Changed the code snippet to match v3.

derberg commented on pages/docs/guides/message-validation.md:
```diff
@@ -0,0 +1,105 @@
+        payload:
+          schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+          schema:
+            $ref: 'https://example.europe-west3.gcp.confluent.cloud/subjects/test/versions/1/schema'
```
```suggestion
asyncapi: 3.0.0
info:
  title: Example with Avro
  version: 0.1.0

channels:
  example:
    address: 'example'
    messages:
      avroMessage:
        payload:
          schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
          schema:
            $ref: 'https://raw.githubusercontent.com/asyncapi/website/20a31a0396b41dd24b1bac877ab7ce3f58037c28/public/resources/casestudies/adeo/CostingRequestPayload.avsc'

operations:
  onMessage:
    action: receive
    channel:
      $ref: '#/channels/example'
```

all would be good but after your change, the example was missing an operation so my code snipped:
- add operation
- update the schema link to a link that actually works and anyone can render this document in studio

derberg replied:
you do not have to do anything on local, if you agree just click commit suggestion
![Screenshot 2023-11-08 at 17 12 34](https://github.com/asyncapi/website/assets/6995927/45ed3758-d24e-40ca-abe1-8fe850b65b55)

animeshk923 replied:
> you do not have to do anything on local, if you agree just click commit suggestion ![Screenshot 2023-11-08 at 17 12 34](https://user-images.githubusercontent.com/6995927/281469471-45ed3758-d24e-40ca-abe1-8fe850b65b55.png)

Thank you for the clarification.

animeshk923 replied:
resolved with bac086f

derberg reviewed (commented):
@AnimeshKumar923 thanks a 💯 mate. I thought I will have to do all these PRs. Just please update code sample


@alequetzalli please review, below you can see in a diff tool I used that previous guide was re-added correctly and only code sample was modified. I also read the guide and no other changes need to happen

![Screenshot 2023-11-08 at 16 05 54](https://github.com/asyncapi/website/assets/6995927/686a9246-56bb-4335-9026-021373192570)

animeshk923 committed:
Update pages/docs/guides/message-validation.md

applied suggestion from: https://github.com/asyncapi/website/pull/2282#discussion_r1386770173

Co-authored-by: Lukasz Gornicki <lpgornicki@gmail.com>

animeshk923 commented:
> @AnimeshKumar923 thanks a 💯 mate. I thought I will have to do all these PRs. Just please update code sample

Just did what I was able to did. Glad to help! :grin: 
Let me know if there is any more issues I can help with, I'll try that...

quetzalliwrites commented:
/rtm
